tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/RabbitMQ/RabbitMQIntegration.cs

```diff
@@ -126,7 +154,14 @@ internal static void SetDataStreamsCheckpointOnConsume(Tracer tracer, Span span,
                 var edgeTags = new[] { "direction:in", $"topic:{tags.Queue ?? tags.RoutingKey}", "type:rabbitmq" };
                 var pathwayContext = dataStreamsManager.ExtractPathwayContext(headersAdapter);
                 span.Context.MergePathwayContext(pathwayContext);
-                span.SetDataStreamsCheckpoint(dataStreamsManager, CheckpointKind.Consume, edgeTags);
+                // size of headers is ignored for now
+                span.SetDataStreamsCheckpoint(
+                    dataStreamsManager,
+                    CheckpointKind.Consume,
+                    edgeTags,
+                    GetHeadersSize(headers) + messageSize,
+                    messageTimestamp);
+                // basicProperties.Timestamp == null ? 0 : DateTimeOffset.UtcNow.ToUnixTimeMilliseconds() - basicProperties.Timestamp.UnixTime
```

Comment:
We can probably remove these comments now?
```suggestion
                span.SetDataStreamsCheckpoint(
                    dataStreamsManager,
                    CheckpointKind.Consume,
                    edgeTags,
                    GetHeadersSize(headers) + messageSize,
                    messageTimestamp);
```